### ISSUE TITLE
Select i.e. archive pages

### ISSUE THREAD

**Issue by sebask** [CMB2/CMB2-Post-Search-field#25]
Currently it's not possible to select "pages" like an post archive page. Any way to accomplish this?

**Comment by jtsternberg** [comment]
Sure it is, use the `'post_type'` field parameter like in the example in the readme, but replace `'post' ` with `'page'`.

**Comment by sebask** [comment]
Sorry for not describing this more clearly in my initial post. I meant an archive page as in the "system" archive page for i.e.: posts (`archive.php)`, or CPT (`archive-{post_type}.php`).

I guess the problem is these are not really registered with an post id.

**Comment by tw2113** [comment]
If I'm following correctly, those would just be the template files chosen to display the results of the query made for that archive. Not quite sure how they'd tie into this search field extension in this case.

**Comment by clovepod** [comment]
The creative solution we've come up with to this issue is to create link format pages that are basically a redirect over to the archive page URL.  We can then select the link page and it acts like a pointer over to the CPT archive page.

**Comment by tw2113** [comment]
Ahhh, @clovepod clued me in on what @sebask was looking for. They wanted to be able to select archive URL from the dropdown.

**Comment by sebask** [comment]
Indeed, we often use those pages to display all post in our custom post types. Btw, Wordpress does show these archive pages in the Menu selector (see screenshot). Maybe that could be useful.
<img width="302" alt="screen shot 2017-05-26 at 13 09 11" src="https://cloud.githubusercontent.com/assets/2178774/26492574/1f9ea93a-4215-11e7-950d-4a369cdcf397.png">

@clovepod Thanks! Sounds like a great quick fix.

**Comment by tw2113** [comment]
The biggest hickup I could potentially see is that the field returns IDs, not archive urls. May get a bit hairy trying to juggle between the display types and also saving of the intended option between the two possible values.
